### DISCUSSION TITLE
Can you default repo settings on each new repo?

### DISCUSSION THREAD

**Discussion by hkdatalore** [gogs/gogs#6879] [Q&A] [answered]
Most of our repos are private and we do not use the wiki.  It would be useful if we could default these options on all new repos that are created.  By default they are public and the wiki is turned on.

**Comment by unknwon** [comment, marked as answer]
Hey, it is currently not possible without changing the code, but please feel free to submit a feature request!

**Reply by hkdatalore** [reply]
Ok, I submitted it.  Thank you!
